### PR TITLE
Add column Description to Portfolio table

### PR DESCRIPTION
**Pull request by jasonburkert-ccpo**
New column is needed for upcoming work.
- Adds column `Description` to `Portfolio` table
- Updates business rule `Portfolio - Last updated changes` to reflect column changes - related to BR added in #167

In addition to the added column, improve documented changes introduced in #293
- Updates Relational Model Diagram
  - adds `Portfolio.Description`
  - marks `Operator.Portfolio` as inactive
  - clarifies relationship between `Portfolio` and `Environment`
  - clarifies relationships between `Environment` and `Operator`
  - clarifies relationships between `Portfolio` and `sys_user`

The recent addition of `Environment` is prominent enough to be included on the simplified ERD.  Splitting this diagram coincides with internal decisions re: value streams.

- Updates Entity-Relationship Diagram (ERD)
  - splits ERD into separate **DAPPS** and **ATAT** diagrams
  - adds `Environment` entity
  - adjusts for current understanding and clarity

Updated diagrams are rendered by GitHub [here](https://github.com/dod-ccpo/atat-snow/blob/73afa06b3fc182c1db51a93b06c8c685b715013c/diagrams.md).

### CODE REVIEW

**Comment by jasonburkert-ccpo** [comment]
### New column on internal form

image

**Comment by jasonburkert-ccpo** [comment]
### Diagram updates
Showing only relevant parts.  New column highlighted.
image

**Comment by jasonburkert-ccpo** [comment]
### Entity-Relationship Diagram (ERD) updates

## DAPPS ERD

image

## ATAT ERD

image

File: f600233d1b154d507b782f84604bcb12/update/sys_script_6481d45d976611106fa8b4b3f153af7f.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
         <change_fields>false</change_fields>
         <client_callable>false</client_callable>
         <collection>x_g_dis_atat_portfolio</collection>
-        <condition>current.name.changes() || current.portfolio_managers.changes() || current.portfolio_viewers.changes() || current.operators.changes()</condition>
+        <condition>current.name.changes() || current.description.changes() || current.active_task_order.changes() || current.portfolio_managers.changes() || current.portfolio_viewers.changes()</condition>
```
Comment (juliusfitzhugh-ccpo) [inline f600233d1b154d507b782f84604bcb12/update/sys_script_6481d45d976611106fa8b4b3f153af7f.xml:15]: Nice, I feel like the BRs get left behind or forgotten at times.

**Review by juliusfitzhugh-ccpo — APPROVED** [review]
Looks good.

File: diagrams.md [hunk 2]
```diff
@@ -1,30 +1,36 @@
-### Entity-Relationship Diagram (ERD)
+### DAPPS Entity-Relationship Diagram (ERD)
```
[the remark this answers is not in the document]
Comment (jasonburkert-ccpo) [reply diagrams.md:1]: It made so much more sense once I tried it.

**Pull request merged**